internal/query/projection/session.go

```diff
@@ -30,6 +30,7 @@ const (
 	SessionColumnIntentCheckedAt      = "intent_checked_at"
 	SessionColumnWebAuthNCheckedAt    = "webauthn_checked_at"
 	SessionColumnWebAuthNUserVerified = "webauthn_user_verified"
+	SessionColumnTOTPCheckedAt        = "totp_checked_at"
```

Comment:
please increase the projection version, so the column will get created on existing databases

Reply:
3 to 4 was done already in https://github.com/zitadel/zitadel/pull/6339 as part of this sprint.